### PR TITLE
feature test UI coin detail screen

### PR DESCRIPTION
**Pull request by tamdev3241**

### CODE REVIEW

File: lib/src/screens/coin_detail_screen/coin_detail_screen.dart [hunk 1]
```diff
@@ -33,7 +33,7 @@ class _CoinDetailScreenState extends State<CoinDetailScreen> {
               color: Colors.black,
               size: 30.0,
             ),
-            onPressed: () {},
+            onPressed: () => Navigator.pop(context),
```
Comment (thanhphamtrung) [inline lib/src/screens/coin_detail_screen/coin_detail_screen.dart:36]: for safe, you should check if(Navigator.canPop) before Navigator.pop(context)

File: lib/src/screens/coin_detail_screen/pages/price_chart_page.dart [hunk 2]
```diff
@@ -1,7 +1,5 @@
-import 'dart:convert';
-
+import '../temp_data/temp_coin_detail_data.dart';
 import 'package:flutter/material.dart';
-import 'package:flutter/services.dart';
 import 'package:font_awesome_flutter/font_awesome_flutter.dart';
 
 import '../../../constants/app_theme.dart';
```
Comment (thanhphamtrung) [inline lib/src/screens/coin_detail_screen/pages/price_chart_page.dart:5]: sort

File: coverage/lcov.info [hunk 3]
```diff
@@ -0,0 +1,238 @@
+SF:lib\src\screens\coin_detail_screen\pages\coin_info_page.dart
```
[the remark this answers is not in the document]
Comment (tamdev3241) [reply coverage/lcov.info:1]: I added it into gitignore.

File: lib/src/screens/coin_detail_screen/coin_detail_screen.dart (hunk 1 above)
Comment (tamdev3241) [reply lib/src/screens/coin_detail_screen/coin_detail_screen.dart]: I added Navigator.canPop() to check can back before back the screen.

File: lib/src/screens/coin_detail_screen/pages/price_chart_page.dart (hunk 2 above)
Comment (tamdev3241) [reply lib/src/screens/coin_detail_screen/pages/price_chart_page.dart]: I fixed it

File: lib/src/screens/coin_detail_screen/coin_detail_screen.dart [hunk 4]
```diff
@@ -1,12 +1,14 @@
+import 'package:flutter/cupertino.dart';
```
Comment (thanhphamtrung) [inline lib/src/screens/coin_detail_screen/coin_detail_screen.dart:1]: cupertino for what?

File: lib/src/screens/coin_detail_screen/pages/price_chart_page.dart [hunk 5]
```diff
@@ -106,61 +90,57 @@ class _PriceChartPageState extends State<PriceChartPage>
...
-                    child: Row(
-                      children: [
-                        SizedBox(
-                          height: 25.0,
-                          width: 25.0,
+                ? Row(
+                    children: [
+                      SizedBox(
+                        height: 25.0,
+                        width: 25.0,
```
Comment (thanhphamtrung) [inline lib/src/screens/coin_detail_screen/pages/price_chart_page.dart:115]: layout variable

**Review by thanhphamtrung — COMMENTED** [review]
layout variable 
cupertino
=> 20k

File: lib/src/screens/coin_detail_screen/pages/price_chart_page.dart (hunk 5 above)
Comment (tamdev3241) [reply lib/src/screens/coin_detail_screen/pages/price_chart_page.dart:115]: I created layout variable into screen

File: lib/src/screens/coin_detail_screen/coin_detail_screen.dart (hunk 4 above)
Comment (tamdev3241) [reply lib/src/screens/coin_detail_screen/coin_detail_screen.dart]: i fixed it

**Review by thanhphamtrung — APPROVED** [review]
lgtm

**Review by thanhphamtrung — COMMENTED** [review]
LGTM

**Pull request merged**